editor/src/components/editor/remix-navigation-bar.tsx

```diff
@@ -45,6 +66,53 @@ export const RemixNavigationBar = React.memo(() => {
 
   const label = getRemixUrlFromLocation(navigationControls[EP.toString(activeRemixScene)]?.location)
```

Comment:
we should probably rename this from label to `activeUrl` or `activeRemixUrl`

Reply:
Fixed in 247525474.